core/templates/dev/head/pages/exploration_player/progress_nav_directive.html

```diff
@@ -1,4 +1,4 @@
-<div layout="row" layout-align="space-between center" style="background-color: white;">
+<div layout="row" layout-align="space-between center" style="background-color: white; margin-left: 87px;">
```

Comment:
As sean mentioned, don't forget to add some comments explaining the 87px.

Reply:
Done!